public/sass/base/_base.scss

```diff
@@ -10,7 +10,59 @@ html {
     box-sizing: border-box;
     font-size: 62.5%;
 }
+body{
+    background-color: yellow;
+}
+
+.splash{
+    position: relative;
+    width: 411px;
+    height: 823px;
+    background: radial-gradient(131.27% 136.09% at 54.62% 52.98%, rgba(237, 104, 86, 0.83) 0%, #8F1F1F 100%);
+}
+
+
+.fade-in{
+    position: absolute;
+    width: 115px;
```

Comment:
use rem/vh/vw instead of pixels. This will reduce our effort for developing a responsive design significantly